runtime/src/bank.rs

```diff
@@ -4211,6 +4211,22 @@ impl Bank {
         self.tick_height.fetch_add(1, Relaxed);
     }
 
+    #[cfg(feature = "dev-context-only-utils")]
+    pub fn register_tick_for_test(&self, hash: &Hash) {
+        // currently meaningless wrapper; upcoming pr will make it an actual helper...
+        self.register_tick(hash)
+    }
+
+    #[cfg(feature = "dev-context-only-utils")]
+    pub fn register_default_tick_for_test(&self) {
+        self.register_tick(&Hash::default())
+    }
+
+    #[cfg(feature = "dev-context-only-utils")]
+    pub fn register_unique_tick(&self) {
```

Comment:
`_for_test` is omitted intentionally due to the use by `VoteSimulator` and `solana-ledger-tool`.